.github/workflows/ci-baseline.yml

```diff
@@ -82,45 +52,23 @@ jobs:
           #- { os: ubuntu-latest, image: intel_oneapi-eigen_3.3.7-gtest, mode: Release }
           #- { os: ubuntu-latest, image: intel_oneapi-eigen_3.3.7-gtest, mode: Debug }
 
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on
-    # The type of machine to run the job on
     runs-on: ${{ matrix.config.os }}
-
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idcontainer
-    # A container to run any steps in a job that don't already specify a container.
     container: pressiomodelreduction/${{ matrix.config.image }}
-
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idenv
-    # A map of environment variables that are available to all steps in the job
     env:
       num_cpus: 2
-      pressio_src: /__w/pressio/pressio
-      pressio_build: /out/pressio/build
       eigen_inc_dir: /usr/local/eigen/install/include/eigen3
       gtest_dir: /usr/local/gtest/install
       small: small
       medium: medium
       large: large
 
-    ############################## STEPS in `CI` job ##############################
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idsteps
-    # A job contains a sequence of tasks called steps.
     steps:
+      - uses: actions/checkout@v3
 
-      # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsuses
-      # Selects an action to run as part of a step in your job. An action is a reusable unit of code.
-      - uses: actions/checkout@v2
-
-      # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsname
-      # A name for your step to display on GitHub.
       - name: configure
-
-        # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsrun
-        # Runs command-line programs using the operating system's shell.
-
         # In this stage pressio-builder script is used for configuration purposes
         run: |
-          cmake -S $pressio_src -B $pressio_build \
+          cmake -B builddir \
             -D CMAKE_BUILD_TYPE:STRING=${{ matrix.config.mode }} \
             -D CMAKE_VERBOSE_MAKEFILE:BOOL=ON \
             -D CMAKE_C_COMPILER=$CC \
```

Comment:
can you please add a short comment to say why we don't specify the source ? i guess it is implied by the checkoutv3 thing?

Reply:
`checkout` action clones the repo into `$GITHUB_WORKSPACE` (this was actually the case for v2 as well) which is the default working directory for all steps. There's no need to use hardcoded paths since cmake will use current directory for source and can handle relative paths for build / testing.